protocol-units/bridge/contracts/src/NativeBridge.sol

```diff
@@ -120,7 +131,23 @@ contract NativeBridge is AccessControlUpgradeable, PausableUpgradeable, INativeB
         emit BridgeTransferCompleted(bridgeTransferId, initiator, recipient, amount, nonce);
     }
 
+    function setRateLimits(uint256 _outboundRateLimit, uint256 _incomingRateLimit) external onlyRole(DEFAULT_ADMIN_ROLE) {
```

Comment:
Wouldn't we want to set both volume and duration? Here we just set volume.

Reply:
Ok, I think this is fine to be restricted to a daily limit.

Reply:
so, we could check if the time constraint has passed since the last transaction has been done, I can test both scenarios if you'd like. It would be nice to test which one is less expensive.

Reply:
I would not call this correct w.r.t. to [MIP-56](https://github.com/movementlabsxyz/MIP/blob/mip/rate-limiter/MIP/MIP-56/README.md).

The amount insured is actually fixed by whatever account. The parameters which can be set are the `risk_period` and the `weight`.

Reply:
This is also still does not use an actual balance to compute the rate limit as was suggested elsewhere.